backend/src/main/scala/cromwell/backend/async/AsyncBackendJobExecutionActor.scala

```diff
@@ -48,38 +48,36 @@ trait AsyncBackendJobExecutionActor { this: Actor with ActorLogging =>
     * If the `work` `Future` completes successfully, perform the `onSuccess` work, otherwise schedule
     * the execution of the `onFailure` work using an exponential backoff.
     */
-  def withRetry(work: Future[ExecutionHandle], onSuccess: ExecutionHandle => Unit, onFailure: => Unit): Unit = {
+  def withRetry(actorContext: ActorContext, work: Future[ExecutionHandle], onSuccess: ExecutionHandle => Unit, onFailure: => Unit)(implicit ec: ExecutionContext): Unit = {
```

Comment:
Why is this not using the real retry?

I know the answer is "cuz it's already there" but if we're fixing it anyways ...